### PR TITLE
Update Eventing 2.0 docs

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

First draft of Kyma 2.0 Eventing doc

Changes proposed in this pull request:

- Adapt docs structure for Kyma 2.0
- Add tutorial on in-cluster eventing

### CODE REVIEW

File: docs/01-overview/02-main-areas/eventing/evnt-01-overview.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Overview
+---
+
+Eventing in Kyma can be used to send and receive events from applications. For example, you can subscribe to events from an external application, and when the user performs an action there, you can trigger your Function or microservice. To subscribe to events, you need to use the Kyma [Subscription CRD](../technical-reference/subtab_customresources/eventing-event-subscription.md).
```
Comment (NHingerl) [inline docs/01-overview/02-main-areas/eventing/evnt-01-overview.md:5]: "can it be used" or is it used? Are there other purposes than sending/receiving events?

File: docs/01-overview/02-main-areas/eventing/evnt-02-useful-links.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: Useful links
+---
+
+You can find out more about Eventing by following the links on this page.
```
Comment (NHingerl) [inline docs/01-overview/02-main-areas/eventing/evnt-02-useful-links.md:5]:
```suggestion
Find out more about Eventing by following the links on this page.
```

File: docs/tutorials/tut-send-event-without-kyma-app.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Prerequisites
+
+- A running Kyma cluster with a valid Namespace
+
+## Steps
+
+1. In Kyma's left navigation panel, go to **Workloads** > **Functions** and navigate to your Function.
+
+2. Once in the Function details view, Switch to the **Configuration** tab, and select **Create Event Subscription** in the **Event Subscriptions** section.
```
Comment (NHingerl) [inline docs/tutorials/tut-send-event-without-kyma-app.md:17]:
```suggestion
2. Once in the Function details view, switch to the **Configuration** tab, and select **Create Event Subscription** in the **Event Subscriptions** section.
```

File: docs/tutorials/tut-send-event-without-kyma-app.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  protocol: ""
+  protocolsettings: {}
+  sink: http://myservice.mynamespace.svc.cluster.local
+
+```
+
+
+4. Select the event type and version that you want to use for your Function and select **Save** to confirm changes.
+
+The message appears on the UI confirming that the Event Subscription was successfully created, and you will see it in the **Event Subscriptions** section in your Function.
```
Comment (NHingerl) [inline docs/tutorials/tut-send-event-without-kyma-app.md:47]:
```suggestion
You get a confirmation that the Event Subscription was successfully created, and you will see it in the **Event Subscriptions** section in your Function.
```

"...you will see **_it_**..." - will I see the message that appears or the Event Subscription? Context implies the subscription, but could be the confirmation message too, sentence is ambiguous.

File: docs/01-overview/02-main-areas/eventing/evnt-01-overview.md (hunk 1 above)
Comment (alexandra-simeonova) [reply docs/01-overview/02-main-areas/eventing/evnt-01-overview.md:5]: You're right, "is used" would be better 🙂

**Review by NHingerl — APPROVED** [review]

File: docs/tutorials/tut-send-event-without-kyma-app.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+title: Send events without a Kyma Application
+type: Tutorials
```
[the remark this answers is not in the document]
Comment (klaudiagrz) [reply docs/tutorials/tut-send-event-without-kyma-app.md:3]: Another question... shouldn't the doc be placed in the `tutorials/eventing` folder? We agreed that we're going to divide docs into areas in the tutorials folder.

[the remark this answers is not in the document]
Comment (alexandra-simeonova) [reply docs/tutorials/tut-send-event-without-kyma-app.md]: Thanks a lot! I moved the tutorial to the new folder, but it looks like I forgot to delete it from the old one 😮  https://github.com/kyma-project/kyma/pull/11309/files#diff-8d703f1471086ed101d48d2b70154c87464fee863dd56d26b7c059f74461992eL3

File: docs/05-technical-reference/03-architecture/evnt-01-architecture.md [hunk 6]
```diff
@@ -3,7 +3,9 @@ title: Eventing Architecture
 type: Architecture
```
Comment (klaudiagrz) [inline docs/05-technical-reference/03-architecture/evnt-01-architecture.md:3]: We don't need the `type` metadata here.

File: docs/05-technical-reference/03-architecture/evnt-01-architecture.md [hunk 7]
```diff
@@ -3,7 +3,9 @@ title: Eventing Architecture
...
 ---
 
-Eventing uses NATS to implement the Event Publisher Proxy and the Eventing Controller which work together to process and deliver events in Kyma.
+Eventing uses NATS to implement the Event Publisher Proxy and the Eventing Controller which work together to process and deliver events in Kyma. See how Eventing works inside a Kyma cluster:
+
+![Eventing architecture](./assets/evnt-architecture.svg)
```
Comment (klaudiagrz) [inline docs/05-technical-reference/03-architecture/evnt-01-architecture.md:8]: Please make the connectors and outlines more visible (use HEX mentioned in the [draft](https://github.com/kyma-project/community/blob/4e619fac18aadb1005927f1a5b77b5c0a576b684/guidelines/content-guidelines/02-diagrams.md#connectors) guidelines for diagrams). Also, add a reference key below to explain the difference between the dotted and normal lines. For the future, consider adding the description for the steps.

```
1. Kyma user creates the Application CR.
2. Application Operator watches for Application CRs.
...
and so on
```

File: docs/05-technical-reference/03-architecture/evnt-02-event-processing.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ type: Architecture
 The event processing and delivery flow uses the NATS server to process events and send them to subscribers.
 This diagram explains the event flow in Kyma, from the moment the Application sends an event, to the point when the event triggers the Function.
 
-![Eventing flow](./assets/eventing-flow.svg)
+![Eventing flow](./assets/evnt-processing-flow.svg)
```
Comment (klaudiagrz) [inline docs/05-technical-reference/03-architecture/evnt-02-event-processing.md:8]: Please adjust the colors in the diagram to the guidelines (see the draft [here](https://github.com/kyma-project/community/blob/4e619fac18aadb1005927f1a5b77b5c0a576b684/guidelines/content-guidelines/02-diagrams.md#background)). Also, the `type` metadata is not needed here.

File: docs/05-technical-reference/03-architecture/evnt-02-event-processing.md [hunk 9]
```diff
@@ -5,7 +5,7 @@ type: Architecture
...
 
 1. The Eventing Controller watches the Subscription Custom Resource. It determines if there are any new incoming events.
```
Comment (klaudiagrz) [inline docs/05-technical-reference/03-architecture/evnt-02-event-processing.md:10]: In point no.3 you've got a link that will not work. Remember to change it after @majakurcius adds the Application Connector docs.

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**